elixir/apps/web/lib/web/live/settings/identity_providers/new.ex

```diff
@@ -84,7 +84,7 @@ defmodule Web.Settings.IdentityProviders.New do
       adapter={@adapter}
       account={@account}
       name="OpenID Connect"
-      description="Authenticate users with a generic OpenID Connect adapter and synchronize users and groups with just-in-time provisioning."
+      description="Authenticate users with a universal OpenID Connect adapter and synchronize users just-in-time provisioning."
```

Comment:
What do you think about removing `provisioning` at the end of this sentence?

Reply:
👍